packages/bygger/src/Forms/formBuilderHooks/beforeSaveComponentSettings.ts

```diff
@@ -10,6 +10,10 @@ const beforeSaveComponentSettings = (submissionData: SubmissionData) => {
       }
     });
   }
+  if (content && typeof content === 'string') {
+    const hrefTrimSpaceRegex = /\shref\s*=\s*/g;
```

Comment:
Hvorfor sjekker du på \s før href? Er det ikke bedre å sjekke uten for å slippe space i replace?

Ett annet alternativ til denne regexen er /?<=href)\s*=\s*/ fordi da trenger man bare replace med '=', så personlig foretrekker jeg den siden den viser tydligere at man bare ønsker å fjerne space rundt =